components/File/src/File.tsx

```diff
@@ -23,11 +24,12 @@ export const File = ({ name, link, size, lastUpdated, className }: FileProps) =>
 
   return (
     <a
-      href={link}
+      href={href}
       className={clsx('denhaag-file', className)}
       aria-labelledby="name"
       aria-describedby="description"
       download={name}
+      onClick={onClick}
```

Comment:
Eigenlijk wil je geen onclick op een anchor tag. Hier zouden we een button van moeten maken als er een onclick event meegegeven wordt.